pre-cache/precache.sh

```diff
@@ -5,6 +5,8 @@ set -e
 rm -rf /host/tmp/precache
 cp -a /opt/precache /host/tmp/
 cp -rf /etc/config /host/tmp/precache/config
+/opt/precache/check_space
+[ $? -ne 0 ] && echo "not enough space for precaching" && exit 17
```

Comment:
There doesn't seem to be a way to pass the return code back to the job. The error will be reported as a generic error in the CGU. @vitus133 @imiller0 @browsell Any objection or alternative? Splitting this into its own job seems to be a bit too much for just a better error message?

Reply:
Are there meaningful log messages that can be looked at to determine the failure reason?

Reply:
yes but you have to look at precache pod logs